test/spec/utils.js

```diff
@@ -0,0 +1,16 @@
+exports.getSearchCount = function*() {
+  var value = yield browser
```

Comment:
'yield' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'browser' is not defined.